plugin/someteam.example.com/v1/validator/validator_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package main_test
+
+import (
+	"runtime"
+	"strings"
+	"testing"
+
+	"sigs.k8s.io/kustomize/pkg/kusttest"
+	"sigs.k8s.io/kustomize/plugin"
+)
+
+func TestValidatorHappy(t *testing.T) {
+	tc := plugin.NewEnvForTest(t).Set()
+	defer tc.Reset()
+
+	tc.BuildExecPlugin("someteam.example.com", "v1", "Validator")
+	path := "kubeval-" + runtime.GOOS
+	tc.CopyFile("someteam.example.com", "v1", "Validator", path, "kubeval")
```

Comment:
just assume its on the $PATH
We cannot check it in.

Reply:
removed